lib/Backend/ServerScriptContext.cpp

```diff
@@ -63,11 +63,15 @@ ServerScriptContext::~ServerScriptContext()
 #endif
     if (m_asmJsInterpreterThunkBufferManager)
     {
+        m_asmJsInterpreterThunkBufferManager->Decommit();
```

Comment:
just double checked, the delete will free first and then fill debugBreak, then decommit. however at this stage we don't need to free the heap and fill debugbreak, so having decommit first should have a little better perf (save a WriteProcessMemory call)

---
In reply to: [85840452](https://github.com/Microsoft/ChakraCore/pull/1846#discussion_r85840452) [](ancestors = 85840452,85839883)